frontend/packages/erd-core/src/stores/userEditing/context.ts

```diff
@@ -7,6 +7,8 @@ export type UserEditingContextValue = {
   activeTableName: string | null
   setActiveTableName: (tableName: string | null) => void
 
+  focusedElementId: string
```

Comment:
I made this to export the hash value, but not a hash updating function.
Currently, the hash will be updated by href of anchor elements, so it's no need to export such an update function.